index.js

```diff
@@ -0,0 +1,5 @@
+const server = require('./api/server');
+const port = 3300;
+server.listen(port, function() {
+  console.log(`\n=== Web API Listening on http://localhost:${port} ===\n`);
+});
```

Comment:
one of the cleanest index.js files I've ever seen.  Great job importing and using separate files.